src/DatePicker/__tests__/DatePicker.test.js

```diff
@@ -4,9 +4,14 @@ import { shallow } from 'enzyme';
 import createWithTheme from '../../../utils/createWithTheme';
 import DatePicker from '../DatePicker';
 
+// Explicitly set the value of today so tests don't fail every day at midnight
+const modifiers = {
+  today: new Date(0)
```

Comment:
What do you think of setting this to `2019-03-11` to preserve the snapshot?

Related to my other question on the PR around the snapshot failing again when the month rolls over.